gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/ZipEntryByteSource.java

```diff
@@ -0,0 +1,19 @@
+package edu.isi.vista.gigawordIndexer;
+
+import com.google.common.io.ByteSource;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class ZipEntryByteSource extends ByteSource {
```

Comment:
`ByteSources` are not meant to wrap streams.  In this case, you can just read in the bytes and use `ByteSource.wrap(bytes).asCharSource()`

Reply:
Modified.